### ISSUE TITLE
Remove Rackspace and OpenStack integrations

### ISSUE THREAD

**Issue by escopecz** [mautic/mautic#8268]
If you run composer install you'll get this warning at the end:

`Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.`

It's because [the library we use](https://github.com/rackspace/php-opencloud) is outdated and we should use its successor:

https://github.com/php-opencloud/openstack

I can see it used only in one plugin: MauticCloudStorageBundle

**Comment by escopecz** [comment]
So there is no replacement for the [rackspace/php-opencloud](https://github.com/rackspace/php-opencloud) library. Only the Openstack library has a [successor library](https://github.com/php-opencloud/openstack). So if we want to support Rackspace we will still have dependency on unsupported library. There is [open PR to update the library from 2016](https://github.com/rackspace/php-opencloud/pull/694) 
I talked to Rackspace support and they want to talk about it but I don't think more talking would do anything. Since many people asked for the merge or any other maintenance of that library for years.

So I see 2 options:
1. We'll use some fork of that library that uses supported dependencies.
2. We'll remove support for Rackspace from Mautic.

I would like to know how many Mautic instances use that integration and decide based on that knowledge. I [asked in the community](https://mautic.slack.com/archives/C8B89CLSF/p1578042090013100).

**Comment by escopecz** [comment]
I made a [poll at the community Slack](https://mautic.slack.com/archives/C8B89CLSF/p1578042090013100) if anyone uses these 2 integrations and so far no one answered that they do. We also scanned all Mautic instances in our cloud where only 1 customer had it enabled but when we tested it it did not even load the list of files. So it may not even work anyway.

It seems no one will miss those 2 integrations. Let's remove them.
